autoware_lanelet2_extension/docs/extra_lanelet_subtypes.md

```diff
@@ -43,3 +43,29 @@ Sample pedestrian lane in .osm format is shown below:
     <tag k="one_way" v="yes"/>
   </relation>
 ```
+
+## Bicycle Lane (format_version >= 2)
+
+Although `bicycle_lane` subtype is already defined in the original [Lanelet2 documentation](https://github.com/fzi-forschungszentrum-informatik/Lanelet2/blob/master/lanelet2_core/doc/LaneletAndAreaTagging.md#subtype-and-location), Autoware compatible with `format_version>=2` supports `bicycle_lane` at component level.
+
+- refers: `road` lanelet adjacent to the objective `bicycle_lane` should share the same boundary LineString
+- specification:
+  - A `bicycle_lane` is connected to/from only `bicycle_lane` lanelets and is adjacent to a `road` lanelet. Namely in Left(Right) hand side traffic rules, it is on the left(right) side of `road` lanelet.
```

Comment:
Shouldn't the first item go under specification? What does `refers` mean in this context?

Reply:
Other sections like "Road shoulder subtype" and "Pedestrian lane subtype" have the same "refer" description so I followed then. But yeah they are ambiguous.